File: Modul-04/SOAL 5/PRAK405-2310817210009-MuhammadIbnuSina.py
X, Z = map(int, input().split())
TOTALKESELURUHAN = 0
PERHITUNGANBARIS = 0

for i in range(1, X + 1):
    for j in range(i, 0, -1):
        if j > 1:
            hasilTunggal = j * Z
            PERHITUNGANBARIS += hasilTunggal
            print(f"({j} * {Z}) + ", end="")
        elif j == 1:
            hasilTerakhir = j * Z
            PERHITUNGANBARIS += hasilTerakhir
            print(f"({j} * {Z}) = {PERHITUNGANBARIS}")
            TOTALKESELURUHAN += PERHITUNGANBARIS
            PERHITUNGANBARIS = 0

print(TOTALKESELURUHAN)

File: Modul-04/SOAL 5/PRAK405-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    int X, Z;
    scanf("%d %d", &X, &Z);
    int TOTAKKESELURUHAN=0, PERHITUNGANBARIS=0;

    for (int I = 1; I < X+1; I++) {
        for (int B = I; B > 0; B--) {
            if (B > 1) {
                int hasilTunggal = (B * Z);
                PERHITUNGANBARIS = PERHITUNGANBARIS + hasilTunggal;
                printf("(%d * %d) + ", B, Z);
            }
            else if (B == 1) {
                int hasilTerakhir = (B * Z);
                PERHITUNGANBARIS = PERHITUNGANBARIS + hasilTerakhir;
                printf("(%d * %d) = %d\n", B, Z, PERHITUNGANBARIS);
                TOTAKKESELURUHAN = TOTAKKESELURUHAN + PERHITUNGANBARIS;
                PERHITUNGANBARIS = 0;
            }
        }
    }
    printf("%d", TOTAKKESELURUHAN);
    return 0;
}